packages/calling-component-bindings/src/handlers/createACSCallHandlers.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Call, CallAgent, StartCallOptions } from '@azure/communication-calling';
+/* @conditional-compile-remove(dialpad) */ /* @conditional-compile-remove(PSTN-calls) */
+/* @conditional-compile-remove(PSTN-calls) */
+import { AddPhoneNumberOptions } from '@azure/communication-calling';
+/* @conditional-compile-remove(PSTN-calls) */
+import {
+  isCommunicationUserIdentifier,
+  isMicrosoftTeamsUserIdentifier,
+  isPhoneNumberIdentifier
+} from '@azure/communication-common';
+import { CommunicationIdentifier } from '@azure/communication-common';
+
+import { _toCommunicationIdentifier } from '@internal/acs-ui-common';
+import { StatefulCallClient, StatefulDeviceManager } from '@internal/calling-stateful-client';
+import memoizeOne from 'memoize-one';
+import { isNonTeamsCallParticipants } from '../utils/callUtils';
+import { CallingHandlersCommon, createDefaultCallingHandlersCommon } from './createHandlersCommon';
+
+/**
+ * Object containing all the handlers required for calling components.
+ *
+ * Calling related components from this package are able to pick out relevant handlers from this object.
+ * See {@link useHandlers} and {@link usePropsFor}.
+ *
+ * @public
+ */
+export type CallingHandlers = Omit<CallingHandlersCommon, 'onStartCall'> & {
+  onStartCall: (participants: CommunicationIdentifier[], options?: StartCallOptions) => Call | undefined;
```

Comment:
why doesn't this support all the options it previously supported:
![image](https://user-images.githubusercontent.com/2684369/200396139-d86e13c0-f59a-4178-ba3e-e38982528d5f.png)

Reply:
Is this a breaking change to reduce the supported parameter set?